common.blocks/pager/pager.css

```diff
@@ -0,0 +1,4 @@
+.pager {
```

Comment:
let's mix pager with `issues__pager` and move these rules as margins to that element